en/qgep_swmm/Create-Input.rst

```diff
@@ -32,8 +32,11 @@ From SWMM GUI you can set the simulation parameters (see `Change Simulation Para
 Python command line
 -------------------
 
+Advanced users might want to run the scripts directly from Python. 
+
 .. code :: python
 
+	from .QgepSwmm import QgepSwmm
```

Comment:
Is this done from the python console?

Reply:
Personally, I launch it from my python outside QGIS, but it could be launched from the the python console within QGIS. Should we keep this python doc or is it confusing for the the qgep swmm users?

Reply:
The relative import `from .QgepSwmm import ...` probably only works if the python interpreter is launched in a very specific environment, so I guess this will be misleading without detailed instructions.

Reply:
Sould I add something like: 

`
import sys
sys.path.append("/path/to/my/plugin/processing_provider/")
`
?

Reply:
Isn't it sufficient that processing itself offers access to this algorithm? I wonder if we should treat the processing algs as "public api" that we announce.
Path mangling is a last resort and not something I'd like to have as "best practice" in our docs.

Reply:
I think you are right, from the comment raised from the user it is more confusing than helping...